index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.length = this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item)
```

Comment:
```suggestion
    this.length = this.items.push(item)
```
Sabes que devuelve el metodo push de los arrays! devuelve el length del array!!! entonces asi lo puedes usar y te ahorras la instruccion del this.length++ que tienes abajo :D